docs/readthedocs/source/doc/Chronos/Howto/how_to_use_benchmark_tool.md

```diff
@@ -0,0 +1,135 @@
+# Use Chronos benchmark tool
+This page demonstrates how to use Chronos benchmark tool to benchmark forecasting performance on platforms.
+
+## Basic Usage
+The benchmark tool is installed automatically when Chronos is installed using conda. To get preliminary knowledge of forecasting performance on platform, run benchmark tool with default options using following command:
+```bash
+benchmark-chronos -l 96 -o 720
+```
+```eval_rst
+.. note::
+    **Required Options**:
+
+     **-l/--lookback** and **-o/--horizon** are required options for Chronos benchmark tool. Use **-l/--lookback** to specify the history time steps while use **-o/--horizon** to specify the output time steps.
```

Comment:
+ for more details, please refer to [here](https://binbin-doc.readthedocs.io/en/benchmark-tool-guide/doc/Chronos/Overview/forecasting.html#regular-regression-rr-style).